components/ScheduleHome.vue

```diff
@@ -1,15 +1,26 @@
 <template>
```

Comment:
This is just me thinking out loud maybe this is stupid thing to ask or assume in Vue-world. 

I know we display this schedule on the home page. But I believe the template will be the same on the schedule page daily view as well.

So do you think we could re-use this component there as well? 

Because if we can I would rename the component to something like: ScheduleDailyList

And then this component can be used on `/` and also on `/schedule/list` too without any modification? At least this is my dream-like scenario. Then even if you switch the days on `/schedule/list` page from today to tomorrow or 2028. 11. 04. we would use this component for rendering that as well just w/ different data.

And you could just add `ScheduleWeeklyList` or `ScheduleDailyGrid` or `ScheduleWeeklyGrid` etc. for the different displays and designs -- maybe there is some `ScheduleAbstract` component that is used by all these more custom components (I'm not sure I know enough about inheritance in Vue-world).

Again, sorry if this is the wrong train of thought but I wanted to mention these ideas.

Reply:
yes, good idea for further developing the structure and UI. for now it is also a good question how and where to show the other days on home page, I would think some dropdown for selecting other days, or scrolling left-right
the other Schedule view is a nice addition to the rework, definitely worth the effort to develop it, now that technically everything is under our fingertips. topic for Saturday!